src/main.c

```diff
@@ -451,6 +451,21 @@ static void test_paths_setup(char **av)
         Scm_AddLoadPath("../../src", FALSE);
         Scm_AddLoadPath("../../libsrc", FALSE);
         Scm_AddLoadPath("../../lib", FALSE);
+    } else {
+        // should be synchronized with the ifdefs in paths to avoid
+        // triggering Scm_Error.
+#if defined(GAUCHE_WINDOWS) || defined(GAUCHE_MACOSX_FRAMEWORK) || defined(__linux__)
```

Comment:
Not so sure about the Mac thing here. It only works if the installation layout is the same as the others, i.e. $(topdir)/bin/gosh.

Reply:
You're right. I think I was confused because in the beginning I return the directory to "bin" instead of the root dir of an installation. I made the change to match with Windows behavior some time later but still forgot about it.

But then Mac still needs to find and return $git when running uninstalled $git/src/gosh. I don't know anything about those framework stuff to be sure that's true.

Reply:
Is `GAUCHE_MACOSX_FRAMEWORK` the usual configuration that people normally use on a Mac, or is it for special circumstances?